src/netex_idf/read.rs

```diff
@@ -45,6 +47,21 @@ where
     stops::from_path(&path.join("arrets.xml"), &mut collections)?;
     // TODO : use _lines_netex_idf to get trips>physical_mode_id
     let _lines_netex_idf = lines::from_path(&path.join("lignes.xml"), &mut collections)?;
+    for offre_folder in WalkDir::new(path)
+        .min_depth(1)
+        .max_depth(1)
+        .into_iter()
+        .filter_map(std::result::Result::ok)
+        .filter(|dir_entry| {
+            dir_entry
+                .metadata()
+                .map(|metadata| metadata.is_dir())
+                .unwrap_or_default()
+        })
+    {
+        info!("Reading offre in folder {:?}", offre_folder);
```

Comment:
for better log `Reading offre in folder DirEntry("input/OFFRE_TOTO")` --> Reading offre in folder input/OFFRE_TOTO`
```suggestion
        info!("Reading offre in folder {:?}", offre_folder.path());
```